src/pinning.js

```diff
@@ -121,10 +124,30 @@ class Pinning {
     this.pubsub = new Pubsub(this.ipfs, ipfsId.id)
     await this.pubsub.subscribe(this.pinningRoom, this._onMessage.bind(this), this._onNewPeer.bind(this))
     this._dbCloseinterval = setInterval(this.checkAndCloseDBs.bind(this), this.dbCheckCloseInterval)
+
+    if (IPFS_METRICS_ENABLED) {
+      // Log out the bandwidth stats periodically
+      this._ipfsMetricsInterval = setInterval(async () => {
+        try {
+          const stats = await this.ipfs.stats.bw()
+          this.logger.info(`Node Bandwidth Stats: ${JSON.stringify(stats)}`)
+
+          const globalStats = this.ipfs.libp2p.metrics.global
+          this.logger.info(`Global Stats: ${JSON.stringify(globalStats)}`)
```

Comment:
What's in the global stats?

Reply:
https://github.com/libp2p/js-libp2p/blob/master/doc/API.md#metricsglobal